build/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM ubuntu
+
+MAINTAINER Luke Skibinski <l.skibinski@elifesciences.org>
+
+# install some basics
+RUN apt-get update
+RUN apt-get install openjdk-8-jre-headless wget git unzip xpdf-utils -y --no-install-recommends
+
+# this will be needed to stop font replacement during squish
+# echo ttf-mscorefonts-installer msttcorefonts/accepted-mscorefonts-eula select true | sudo debconf-set-selections
+# apt-get install ttf-mscorefonts-installer
+
+# clone project
+RUN cd /opt && git clone https://github.com/elifesciences/strip-coverletter/
+WORKDIR /opt/strip-coverletter
+
+# install cajda
+RUN wget --quiet https://github.com/torakiki/sejda/releases/download/v3.2.38/sejda-console-3.2.38-bin.zip
+RUN unzip -q sejda-console-3.2.38-bin.zip && ln -s sejda-console-3.2.38 sejda
+ENV PATH="/opt/strip-coverletter:/opt/strip-coverletter/sejda/bin:${PATH}"
+
+RUN git pull && git checkout --quiet a514bd43512b1a133668008cb8e698b0e408a18e
```

Comment:
is missing a `CMD` statement to run a default script?

Reply:
Dockerfile isn't finished yet unfortunately. Even if it were finished in this PR, I would still need to investigate how the bot is calling the script and update it. 

I got to the point of researching the different types of mounts Docker supports so that I could pass a file in and spit one out, but calling the Docker container [was taking much longer than calling the script directly](https://github.com/docker/for-mac/issues/1183#issuecomment-345899718 ) so I just put it aside for now.